npm/src/lib/libDef.js

```diff
@@ -148,7 +148,8 @@ export async function getLocalLibDefs(
   return libDefs;
 };
 
-const FLOW_DIR_NAME_RE = /^flow-(all|([><]?=)?v([0-9]+)\.([0-9]+|x)\.([0-9]+|x))$/;
+const FLOW_DIR_NAME_RE =
+  /^flow-(all|([><]?=)?v([0-9]+)\.([0-9]+|x)\.([0-9]+|x)(_([><]?=)?v([0-9]+)\.([0-9]+|x)\.([0-9]+|x))?)$/
```

Comment:
Also I guess this leaves `([><]?=)` as optional in the upper bound... Should we make it required?